Tools/ChangeLog

```diff
@@ -1,3 +1,18 @@
+2022-01-21  Jonathan Bedard  <jbedard@apple.com>
+
+        [webkitscmpy] Stop killing finished processes
+        https://bugs.webkit.org/show_bug.cgi?id=235427
+        <rdar://problem/87887233>
+
+        Reviewed by NOBODY (OOPS!).
+
+        * Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py:
+        (Git.Cache.populate): Check if process has finished before killing it.
```

Comment:
Does it solve the problem of "No such process"?

Reply:
It _should_, but it's not a problem I've been able to reproduce. It's the only way I can see us getting into that codepath, though. It's likely to be a pretty rare and race-y failure as is.